berdlhub/config/profiles.py

```diff
@@ -10,37 +10,65 @@ def configure_profiles(c):
 
     c.KubeSpawner.profile_list = [
         {
-            "display_name": "Small Server (2G RAM, 1 CPU)",
-            "description": "Suitable for light analysis and development",
+            "display_name": "Small: Spark 1 Worker (2GB, 1 core) + Master (1GB, 1 core) - Notebook (2GB, 1 core)",
+            "description": "Minimal Spark cluster for light analysis and development",
+            "slug": "small",
             "default": True,
             "kubespawner_override": {
                 "mem_limit": "2G",
                 "mem_guarantee": "1G",
                 "cpu_limit": 1,
                 "cpu_guarantee": 0.5,
                 "image": berdl_image,
+                "environment": {
+                    "DEFAULT_WORKER_COUNT": "1",
```

Comment:
I don't think the notebook needs these env vars.

Reply:
the notebook can use them to set the spark executor/driver memory/cores.

Reply:
But the notebook doesn't start it up, the prespawn hook does.

Reply:
The spark executor memory is set by the spark-kube service API. 
The driver memory is set via a default in the notebook.

These env vars seem to be for the spark-kube-service API

Reply:
no - spark executor memory is not set by spark-kube service. The service creates a spark clustering to use. But the spark session is set by get_spark_session method, that method need to know how to set the executor memory.